externals/kyuubi-spark-sql-engine/src/main/resources/python/kyuubi_util.py

```diff
@@ -72,16 +62,17 @@ def connect_to_exist_gateway():
 
     return gateway
 
+
 def _get_exist_spark_context(self, jconf):
     """
     Initialize SparkContext in function to allow subclass specific initialization
     """
     return self._jvm.JavaSparkContext(self._jvm.org.apache.spark.SparkContext.getOrCreate(jconf))
 
-def get_spark():
+
+def get_spark_session():
```

Comment:
`spark` `sc` is pre-defined vars in spark-shell, not sure if same on pyspark

Reply:
Only `spark` will be added to global_dict here.
In PySpark, pre-defined vars are added in from `SparkContext._create_shell_session()`.
I prefer change to `_create_shell_session` to get or create Spark session from current thread, instead of using 'local' for master and 'test' for app name here. 
What do you think? @cfmcgrady

Reply:
make sense if it works :)

Reply:
Good, let me switch it to `SparkContext._create_shell_session()`.